vllm/worker/model_runner.py

```diff
@@ -446,51 +454,51 @@ def get_size_or_none(x: Optional[torch.Tensor]):
             py_data = receving_list[0]
             input_tokens = torch.empty(*py_data["input_tokens_size"],
                                        dtype=torch.long,
-                                       device="cuda")
+                                       device=self.device)
```

Comment:
Can we use the `_set_default_torch_device` context manager here to not repeat `device=self.device`?